memory/src/main/java/org/opensearch/ml/memory/action/conversation/UpdateInteractionTransportAction.java

```diff
@@ -62,6 +64,7 @@ protected void doExecute(Task task, ActionRequest request, ActionListener<Update
         String interactionId = updateInteractionRequest.getInteractionId();
         try (ThreadContext.StoredContext context = client.threadPool().getThreadContext().newStoredContext(true)) {
             Map<String, Object> updateContent = updateInteractionRequest.getUpdateContent();
+            updateContent.putIfAbsent(ConversationalIndexConstants.INTERACTIONS_UPDATED_TIME_FIELD, Instant.now());
```

Comment:
nit: do we need to support the case `updateTime` pass through with request body?